src/services/config/config.service.ts

```diff
@@ -37,6 +38,10 @@ export default class ConfigService {
         'MISSING_KEY',
       ENABLE_STABLE_POOLS: process.env.VUE_APP_ENABLE_STABLE_POOLS === 'true',
       PORTIS_DAPP_ID: process.env.VUE_APP_PORTIS_DAPP_ID || 'MISSING_KEY',
+      WALLET_SCREENING:
+        (process.env.VUE_APP_WALLET_SCREENING &&
```

Comment:
This can just be `process.env.VUE_APP_WALLET_SCREENING === 'true'` as if `VUE_APP_WALLET_SCREENING` is not set it will be undefined and thus be false

Reply:
[👍](https://emojipedia.org/thumbs-up/)